ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpSourceSpec.scala

```diff
@@ -90,9 +115,7 @@ trait CommonFtpSourceSpec extends BaseSpec {
       probe.expectComplete()
 
       val expectedNumOfBytes = getLoremIpsum.getBytes().length * numOfFiles
-      val total = result.foldLeft(0L) {
-        case (acc, next) => acc + next.count
-      }
+      val total = result.map(_.count).sum
```

Comment:
Given this is a test, I appreciate increased readability over performance. I am not too adamant about it, though :) I can put it back if there is a strong argument